src/store/customer/actions.js

```diff
@@ -267,6 +268,12 @@ export const applyGiftCard = ({commit}, payload) => {
         data.errors[0].message = 'The Code is invalid! Please try again!'
       }
       _procAlert(data, `$${amount} has just been applied successfully`)
+      if (!data.errors) {
```

Comment:
no need to do this, just change _procAlert :
**export const _procAlert = (data, successMessage) => {
  if (data.errors) _alert(data.errors[0].message, 'warning')
  else if (successMessage&&successMessage !== true) {
    _alert(successMessage, 'positive')
  }
}**